### PR TITLE
Upgrade Kubelet to version 1.14.3 and some minor fixes

### PR DESCRIPTION
**Pull request by mohitevishal**
Upgrade Kubelet to version 1.14.3. Also fix the variables names to be more consistent across repos.
Release notes for Kubernetes 1.14.3 available at https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1143

### CODE REVIEW

**Review by alex-vmw — APPROVED** [review]

File: defaults/main.yml [hunk 1]
```diff
@@ -3,8 +3,12 @@ kube_config_dir: "/etc/kubernetes"
 kube_cert_dir: "/etc/kubernetes/ssl"
 kube_manifests_dir: "/etc/kubernetes/manifests"
 kubernetes_service_addresses: "172.21.0.0/16"
-kubernetes_version: "1.4.1"
-hyperkube: "gcr.io/google_containers/hyperkube:v{{kubernetes_version}}"
+
```
Comment (puneetkatyal) [inline defaults/main.yml]: Using kubelet variables is misleading. I would simply change this to --
```suggestion
kubernetes_version: "1.14.3"
hyperkube_repo: "gcr.io/google_containers"
hyperkube: "{{hyperkube_repo}}/hyperkube:v{{kubernetes_version}}"
```

**Review by puneetkatyal — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (mohitevishal) [reply defaults/main.yml]: Those are naming conventions we use across other repos, we should keep it consistent.
For gcr_repo, we use it for other components along with hyperkube/kubelet.

**Comment by mohitevishal** [comment]
@jinghuaz-vmw @qizh01

File: defaults/main.yml (hunk 1 above)
Comment (mohitevishal) [reply defaults/main.yml]: One second thought made the change since hyperkube though would be at same version as kubelet, it's better to distinguish.

**Pull request merged**